### DISCUSSION TITLE
the transfer of chia coins from the gate. io exchange did not reach the tandem wallet

### DISCUSSION THREAD

**Discussion by Butterflygeorge** [Chia-Network/chia-blockchain#17083] [Support]
Good evening.
2023-12-16 I made two transactions from the exchange gate.io chia tokens to your tandem wallet.
transactions have been completed (Done & Blockchain Pending). They haven't come to my wallet for the second day. (Address xch16y2amd7je9j44u9rp54yvqhwycjm63tpxmf3h8ypks78wyje3mrqrllpya )
The chia scan does not show transactions and returns errors.
I want to clarify where my chia tokens are located if they are not on my exchange account or in my wallet. Please help me with solving this issue.

**Comment by BrandtH22** [comment]
Hey @Butterflygeorge  , there have been no stalls or issues with the blockchain. Currently there is high usage of the chain which is delaying transaction that have been submitted with no or insufficient fees.

I would contact the exchange and have them resubmit the transaction with higher blockchain fees to ensure it is processed in a timely manner.

Let us know if you have any further questions and feel free to join our discord server for more timely and thorough support https://discord.gg/chia

**Reply by Butterflygeorge** [reply]
2023-12-16
23:43:29 Transaction TxID 0xe23abb1848836ca064975015ea399d8d1946cf321845c19dd66bbe426685da74
Amount 1.559564 Commission 0.0147
Done & Blockchain Pending

2023-12-16
22:52:01 Transaction TxID 0x441d5181e773eb1e751819ad79f2afbafe78d234ced8a6486ccf9428f7229b94
Amount 116 Commission 0.0147
Done & Blockchain Pending

**Reply by Butterflygeorge** [reply]
The second day there is no chia

**Reply by BrandtH22** [reply]
Hey @Butterflygeorge , have you contacted the exchange to have them resubmit the transaction with higher fees? 

With the current chain utilization transactions will need fees in order to be processed in a timely manner.

**Comment by zhang808908** [comment]
我从钱包转交易所，3天了一直转不出去，不知道怎么处理。

**Reply by BrandtH22** [reply]
Hey @zhang808908 , with the current chain utilization you will need to make sure that fees are added to any transactions that need to be processed in a timely manner 👍

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
